client/my-sites/plan-features-2023-grid/types.ts

```diff
@@ -23,7 +23,7 @@ export type PlanProperties = {
 	rawPrice: number | null;
 	isMonthlyPlan: boolean;
 	tagline: string;
-	storageOptions: string[];
+	storageOptions: { slug: string; planDefault: boolean; feature: FeatureObject }[];
```

Comment:
Resolved in [5b6fa61](https://github.com/Automattic/wp-calypso/pull/79041/commits/5b6fa6189496c5341c5c55f6303233315b12a54d)

I went with storageOptions since the type will be used in checkout as well as plans